app/views/request/describe_notices/_error_message.html.erb

```diff
@@ -1 +1 @@
- _("Thank you! We'll look into what happened and try and fix it up.")
\ No newline at end of file
+<%= _("Thank you! We'll look into what happened and try and fix it up.") %>
```

Comment:
I think we should drop this from the rails upgrade so that we avoid creating translations